pages/_app.js

```diff
@@ -89,11 +90,31 @@ function MyApp({ Component, pageProps }) {
                   <meta name="viewport" content="initial-scale=1.0, width=device-width" />
                   <link rel="icon" href="/assets/img/w3d-logo-symbol-ac.svg" />
                 </Head>
-                <NavbarComponent />
-                <div className="relative z-10"></div>
-                <Component {...pageProps} />
-                <Footer />
-                <ToastContainer />
+                <div
+                  style={{
+                    position: 'absolute',
+                    top: 0,
+                    background: 'red',
```

Comment:
It seems that the background: red code might have been left in unintentionally. I believe it shouldn't be here.

Reply:
@nomadbitcoin , that was exactly it, sorry for the lack of attention